### ISSUE TITLE
Previous command

### ISSUE THREAD

**Issue by derekperkins** [gorhill/cronexpr#14]
I'd love to be able to step backwards to see the previous time a cron has fired.

**Comment by andreykyz** [comment]
Yes, I'd love to. I need to restart some task if computer was poweroff.
